EIPS/eip-5388.md

```diff
@@ -0,0 +1,151 @@
+---
+eip: 5388
+title: Token-Gated HTTP Endpoints
+description: Composable RESTful and Solidity interface to implement token-gated HTTP endpoints using data tokens.
+author: Tim Daubenschütz (@TimDaub)
+discussions-to: https://ethereum-magicians.org/t/token-gated-https-endpoints/10205
+status: Draft
+type: Standards Track
+category: ERC
+created: 2022-08-01
+requires: 712
+---
+
+## Abstract
+
+This standard introduces composable RESTful HTTP and Solidity interfaces that enable paying to access HTTP endpoints using [EIP-20](./eip-20.md) tokens.
```

Comment:
I am not sure it is about REST?
Also, why require ERC20... I think it could be more flexible...

Reply:
`This standard introduces composable HTTP and Solidity interfaces that enable restricting to access HTTP endpoints using smart contracs.`